src/Workspaces/Remote/Core/Services/AssetStorage.cs

```diff
@@ -18,8 +18,8 @@ internal class AssetStorage
     {
         public static readonly AssetStorage Default = new AssetStorage();
 
-        private const int CleanupInterval = 3; // 3 minutes
-        private const int PurgeAfter = 30; // 30 minutes
+        private const int CleanupInterval = 1; // 1 minutes
+        private const int PurgeAfter = 3; // 3 minutes
```

Comment:
1. These feel like these should absolutely not be constants.
2. We can make them non-constants and use useful types like TimeSpan.FromMinutes(...)
3. 3-minutes seems *super short*.  If i get up and go use the restroom or get a coffee, will i face a total purge/sync the moment i get back?